dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -97,6 +97,27 @@ class LessonsControllerTest < ActionController::TestCase
     @pilot_section = create :section, user: @pilot_teacher, script: @pilot_script
     @pilot_student = create(:follower, section: @pilot_section).student_user
 
+    @pilot_instructor = create :teacher, pilot_experiment: 'pl-my-experiment'
+    @pilot_pl_script = create :script, name: 'pl-pilot-script', pilot_experiment: 'pl-my-experiment', is_migrated: true, include_student_lesson_plans: true
+    pilot_pl_lesson_group = create :lesson_group, script: @pilot_pl_script
+    @pilot_pl_lesson = create(
+      :lesson,
+      script_id: @pilot_pl_script.id,
+      lesson_group: pilot_pl_lesson_group,
+      name: 'Pilot Lesson 1',
+      absolute_position: 1,
+      relative_position: 1,
+      has_lesson_plan: true,
+      lockable: false,
+      properties: {
+        overview: 'lesson overview',
+        student_overview: 'student overview'
```

Comment:
is there anything special about this lesson? if not, can you try generating it using `create :script, :with_lessons` instead? https://github.com/code-dot-org/code-dot-org/blob/fe459f188996dc07056a524ae1891be4219d2aa4/dashboard/test/factories/factories.rb#L761

if you need to add or modify anything after you create it, you can always do that after: https://github.com/code-dot-org/code-dot-org/blob/9ceee45455abeb12601b619d3e102fa780a06024/dashboard/test/controllers/lessons_controller_test.rb#L574-L577

this should make it easier to see if you are just creating a lesson or if there is anything special about it.